book/src/getting-started/6-cargo-executable.md

```diff
@@ -12,6 +12,8 @@ previous example, but without using CMake or another C++ build system. Cargo wil
 just like a typical Rust application. Because cxx-qt does not bind the entire Qt API, we will still
 need to write a bit of C++ code. However, we'll use the cxx-qt-build crate to compile it instead of CMake.
 
+Note that the folder structure of this example is different to the CMake tutorial, this is because the `Cargo.toml` is now in the root. So there isn't a `rust` folder, instead just a `src` and the Rust files have moved "up" one folder.
```

Comment:
```suggestion
Note that the folder structure of this example is different to the CMake tutorial, this is because the `Cargo.toml` is now in the root. So there isn't a `rust` folder, instead just a `src` folder and the .rs files have moved up one folder.
```

Reply:
Done